libs/src/services/creatorNode/CreatorNode.ts

```diff
@@ -442,24 +556,28 @@ class CreatorNode {
 
   /**
    * Syncs a secondary creator node for a given user
-   * @param {string} secondary
-   * @param {string} primary specific primary to use
-   * @param {boolean} immediate whether or not this is a blocking request and handled right away
-   * @param {boolean} validate whether or not to validate the provided secondary is valid
+   * @param secondary
+   * @param primary specific primary to use
+   * @param immediate whether or not this is a blocking request and handled right away
+   * @param validate whether or not to validate the provided secondary is valid
    */
-  async syncSecondary (
-    secondary,
-    primary = null,
+  async syncSecondary(
```

Comment:
Decent amt of subtle (but good) behavioral changes in this fn, special care needed. tagging, @SidSethi , but i don't think blocking